java/code/src/com/redhat/rhn/domain/channel/ChannelFamilyFactory.java

```diff
@@ -243,8 +274,9 @@ public static String getNameByLabel(String label) {
      */
     @SuppressWarnings("unchecked")
     public static List<ChannelFamily> getAllChannelFamilies() {
-        Session session = getSession();
-        Criteria c = session.createCriteria(ChannelFamily.class);
-        return c.list();
+        String sql = "SELECT * FROM rhnChannelFamily";
+        TypedQuery<ChannelFamily> query =
+                getSession().createNativeQuery(sql, ChannelFamily.class);
```

Comment:
```suggestion
        TypedQuery<ChannelFamily> query = getSession().createNativeQuery(sql, ChannelFamily.class);
```
Doesn't it fit in 120 characters?